docs/src/locators.md

```diff
@@ -188,104 +186,98 @@ await page.getByRole('checkbox', { checked: true, name: "Check me" }).check();
 ```
 
 ```python async
-await page.get_by_role("button", name=re.compile("(?i)submit")).click()
+await page.get_by_role("button", name=re.compile("submit", re.IGNORECASE)).click()
 
 await page.get_by_role("checkbox", checked=True, name="Check me"]).check()
 ```
 
 ```python sync
-page.get_by_role("button", name=re.compile("(?i)submit")).click()
+page.get_by_role("button", name=re.compile("submit", re.IGNORECASE)).click()
 
 page.get_by_role("checkbox", checked=True, name="Check me"]).check()
 ```
 
 ```java
-page.getByRole("button", new Page.GetByRoleOptions().setName(Pattern.compile("(?i)submit"))).click();
+page.getByRole("button", new Page.GetByRoleOptions().setName(Pattern.compile("submit", Pattern.CASE_INSENSITIVE))).click();
 
 page.getByRole("checkbox", new Page.GetByRoleOptions().setChecked(true).setName("Check me"))).check();
 ```
 
 ```csharp
-await page.GetByRole("button", new() { Name = new Regex("(?i)submit") }).ClickAsync();
+await page.GetByRole("button", new() { Name = new Regex("submit", RegexOptions.IgnoreCase) }).ClickAsync();
 
 await page.GetByRole("checkbox", new() { Checked = true, Name = "Check me" }).CheckAsync();
 ```
 
-[Learn more about the `role` selector](./selectors.md#role-selector).
+Role locators follow W3C specificaitons for [ARIA role](https://www.w3.org/TR/wai-aria-1.2/#roles), [ARIA attributes](https://www.w3.org/TR/wai-aria-1.2/#aria-attributes) and [accessible name](https://w3c.github.io/accname/#dfn-accessible-name).
+
+Note that role locators **do not replace** accessibility audits and conformance tests, but rather give early feedback about the ARIA guidelines.
 
-### Define explicit contract and use `data-test-id=`
+### Define explicit contract and use [`method: Page.getByTestId`]
 
-User-facing attributes like text or accessible name can change frequently. In this case it is convenient to define explicit test ids, for example with a `data-test-id` attribute. Playwright has dedicated support for `id`, `data-test-id`, `data-test` and `data-testid` attributes.
+User-facing attributes like text or accessible name can change over time. In this case it is convenient to define explicit test ids.
 
 ```html
 <button data-test-id="directions">Itinéraire</button>
```

Comment:
our api docs say: By default, the data-testid attribute is used as a test id. so should we be using data-testid or data-test-id` or are both acceptable?

Reply:
Ouch, my bad!